Source/ACE/Entity/Creature.cs

```diff
@@ -495,5 +497,162 @@ protected virtual void UpdateVitalInternal(CreatureVital vital, uint newVal)
                 new ActionChain(this, () => VitalTickInternal(vital)).EnqueueChain();
             }
         }
+
+        public override void SerializeIdentifyObjectResponse(BinaryWriter writer)
```

Comment:
to be fair, most of the stuff returned in the creatureprofile displayed in the client are bad numbers/stats, so i'm kinda okay with breaking it in one regard.. 

i've got some thoughts on how to implement weenietype and delete debugobject going forward but it'll take some db work + thinking/coding time to propose properly, possibly looking to do that in a week or so